src/lib/html.jasmine.reporter.js

```diff
@@ -265,7 +265,7 @@ jasmineRequire.HtmlReporter = function (j$) {
               title: 'randomized with seed ' + order.seed,
               href: seedHref(order.seed)
             },
-            order.seed
+            String(order.seed)
```

Comment:
```suggestion
            order.seed.toString()
```
An alternative if we're happy to say seed is guaranteed to be defined here. (I believe it will be.)